test/libyul/yulOptimizerTests/fullSuite/medium.yul

```diff
@@ -21,11 +21,10 @@
 // ----
 // {
 //     {
-//         let _1 := 0x40
-//         mstore(_1, add(mload(_1), 0x20))
-//         let p := mload(_1)
-//         mstore(_1, add(p, _1))
-//         mstore(add(p, 96), 2)
-//         mstore(_1, 0x20)
+//         let _1 := mload(0x40)
+//         mstore(0x40, add(_1, 0x20))
+//         mstore(0x40, add(_1, 96))
+//         mstore(add(_1, 128), 2)
```

Comment:
Why did this became 128?

Reply:
The meaning of `_1` changed significantly, I think the code is still correct. Let me try `yulrun`.

Reply:
Yep, both versions store at 128.

Reply:
It used to be ``add(p, 96)``, where ``p _:= mload(_1)`` - and there is ``add(mload(0x40), 0x20)`` stored at ``_1`` at that point - i.e. ``add(p, 96)`` is ``add(mload(0x40), 96+0x20 = 128)`` - and ``mload(0x40)`` is stored in ``_1`` in the new version - so fine!